### PR TITLE
Translate `FAQ > File Structure` page to pt-BR

### PR DESCRIPTION
**Pull request by RodolfoSilva**
#1

### CODE REVIEW

**Comment by RodolfoSilva** [comment]
Yes, sure @cezaraugusto 👍 . I'm going to do it now.

**Comment by RodolfoSilva** [comment]
Done @cezaraugusto

File: content/docs/faq-structure.md [hunk 1]
```diff
@@ -59,16 +59,16 @@ components/
...
 
-#### Don't overthink it {#dont-overthink-it}
+#### Não pense muito {#dont-overthink-it}
 
-If you're just starting a project, [don't spend more than five minutes](https://en.wikipedia.org/wiki/Analysis_paralysis) on choosing a file structure. Pick any of the above approaches (or come up with your own) and start writing code! You'll likely want to rethink it anyway after you've written some real code.
+Se você está apenas começando um projeto, [não gaste mais do que cinco minutos](https://en.wikipedia.org/wiki/Analysis_paralysis) na escolha de uma estrutura de arquivos. Escolha qualquer uma das abordagens acima (ou crie as suas próprias) e comece a escrever o código! Você provavelmente vai querer repensá-lo assim mesmo depois de ter escrito algum código real.
```
Comment (fjoshuajr) [inline content/docs/faq-structure.md]:
```suggestion
Se você está apenas começando um projeto, [não gaste mais do que cinco minutos](https://en.wikipedia.org/wiki/Analysis_paralysis) na escolha de uma estrutura de arquivos. Escolha qualquer uma das abordagens acima (ou crie as suas próprias) e comece a escrever o código! Você provavelmente vai querer repensá-lo de qualquer jeito depois de ter escrito algum código.
```

File: content/docs/faq-structure.md [hunk 2]
```diff
@@ -59,16 +59,16 @@ components/
...
 
-If you feel completely stuck, start by keeping all files in a single folder. Eventually it will grow large enough that you will want to separate some files from the rest. By that time you'll have enough knowledge to tell which files you edit together most often. In general, it is a good idea to keep files that often change together close to each other. This principle is called "colocation".
+Se você se sentir completamente preso, comece mantendo todos os arquivos em uma única pasta. Eventualmente ele crescerá o suficiente para separar alguns arquivos dos demais. A essa altura, você terá conhecimento suficiente para informar quais arquivos são modificados juntos com mais frequência. Em geral é uma boa ideia manter os arquivos que geralmente mudam juntos uns dos outros. Este princípio é chamado de "colocation".
```
Comment (fjoshuajr) [inline content/docs/faq-structure.md]:
```suggestion
Se você se sentir completamente preso, comece mantendo todos os arquivos em uma única pasta. Eventualmente ele crescerá o suficiente para que você deseje separar alguns arquivos dos demais. A essa altura, você terá conhecimento suficiente para saber quais arquivos são modificados juntos com mais frequência. Em geral é uma boa ideia manter os arquivos que geralmente mudam juntos uns dos outros. Este princípio é chamado de "colocation".
```

File: content/docs/faq-structure.md [hunk 3]
```diff
@@ -59,16 +59,16 @@ components/
...
 
-As projects grow larger, they often use a mix of both of the above approaches in practice. So choosing the "right" one in the beginning isn't very important.
+À medida que os projetos vão crescendo, eles costumam usar uma mistura de ambas as abordagens acima na prática. No começo não é muito importante, então escolher o "certo".
```
Comment (fjoshuajr) [inline content/docs/faq-structure.md]:
```suggestion
À medida que os projetos vão crescendo, eles costumam usar uma mistura de ambas as abordagens acima na prática. Então escolher a abordagem "certa" no começo não é muito importante.
```

File: content/docs/faq-structure.md (hunk 1 above)
Comment (RodolfoSilva) [reply content/docs/faq-structure.md]: Done!

File: content/docs/faq-structure.md (hunk 2 above)
Comment (RodolfoSilva) [reply content/docs/faq-structure.md]: Done!

**Comment by RodolfoSilva** [comment]
@fjoshuajr done!, Thanks 😄

**Comment by fjoshuajr** [comment]
> thanks @RodolfoSilva LGTM

Ok, will merge this.

**Pull request merged**